README.md

```diff
@@ -133,6 +133,7 @@ Paths:
 * `/etc/hosts`: optional, if present not a bind mount
 * `/etc/krb5.conf.d`: directory, not a bind mount
 * `/etc/localtime`: optional, if present not a bind mount
+* `/etc/machine-id`: not a bind mount
```

Comment:
I think this is a case where we should rather fix the code than add to the list of requirements in the documentation. :)

Specifically in `mountBind` we should add a branch for `fi.Mode().IsRegular()` and `os.Create()` the `containerPath` there, ignoring the error in case the file already existed before.

Reply:
See https://github.com/containers/toolbox/pull/794